### PR TITLE
Issue 6471 - Introduce new option to skip header and summary

### LINKED ISSUE #6471: Introduce new option to skip header and summary
#6454 reminds me that sometimes we want to skip header and summary entirely from the output (posting examples online for example). Currently the only way to skip headers is to use `-q`, but this is a global option which affects many other parts (like diff and progress reporting).

How about a `--lean-terminal` or something which only skips both header and summary?

### PR DESCRIPTION
**Pull request by piotrhm**
This pull request implements suggested approach for issue #6471.

Fix #6471.

### CODE REVIEW

File: src/_pytest/terminal.py [hunk 1]
```diff
@@ -84,6 +84,20 @@ def pytest_addoption(parser):
         dest="verbose",
         help="increase verbosity.",
     )
+    group._addoption(
```
Comment (symonk) [inline src/_pytest/terminal.py:87]: would using a store_true/store_false be more appropriate here? assuming we expect this only once on the command line (unlike -v) we behaves differently based on the count

**Comment by symonk** [comment]
Hi @piotrhm thanks for working on this.  Could you add some test(s) to cover such scenarios as well if you get a chance? :)

File: src/_pytest/terminal.py (hunk 1 above)
Comment (piotrhm) [reply src/_pytest/terminal.py:87]: done

**Comment by piotrhm** [comment]
@symonk thanks for comments! I added tests and changed ._addoption signature. I will be grateful if you take a look at current code.

**Comment by nicoddemus** [comment]
Sorry, should have squashed this one... 😞

**Pull request merged**